lib/dom_html.ml

```diff
@@ -0,0 +1,727 @@
+module Prop = struct
+  type t = string * Js_of_ocaml.Js.Unsafe.any
+
+  let string key value =
+    (key, Js_of_ocaml.Js.Unsafe.inject (Js_of_ocaml.Js.string value))
+
+  let bool key value =
+    (key, Js_of_ocaml.Js.Unsafe.inject (Js_of_ocaml.Js.bool value))
+
+  let int key (value : int) = (key, Js_of_ocaml.Js.Unsafe.inject value)
+
+  let float key (value : float) = (key, Js_of_ocaml.Js.Unsafe.inject value)
+
+  let event key (f : _ Event.synthetic -> unit) =
```

Comment:
Maybe worth adding some test for `event` in `test_jsoo_react` (it uses `react-dom/test-utils`).

Here's an example of other event handling tests:

https://github.com/ml-in-barcelona/jsoo-react/blob/504cb23a78f84e6ecf64566c197a65bdaee3df4a/test/test_jsoo_react.re#L400

Reply:
I'm actually working on converting the whole test file to OCaml, and have already found a couple issues, with name collisions and lack of support for applying props conditionally. I'll post more details once I'm done.